test/install/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include <snowplow/configuration/emitter_configuration.hpp>
```

Comment:
Just wondering if there is any way we could make use of the example app in `examples/main.cpp` for this purpose? It would just be easier to maintain in the future to only have one such app although I understand that the two apps serve different purposes (one is to showcase the tracking APIs and one to test the build).

Reply:
@tamaskenezlego Happy to merge this regardless of this comment unless you have something else in mind?

Reply:
I'm just trying out how this would work, I'll get back to you soon.

Reply:
Ok, I think it was a good idea, I was able to do it without any compromises. Now `examples/main.cpp` is built both as part of the main project and in standalone mode, in the install test workflow. 679f955a16f49eba563067bf525c57c04cd3fa76
Please review and if it's fine you can merge.

Reply:
This is perfect, so much cleaner! Just a minor thing – could you update the README? It says to run the example app using `./snowplow-example {{ your collector uri }}` from inside the build folder, but now I think it is `./examples/snowplow-example {{ your collector uri }}`

Reply:
You're right, fixed it! a1439d3a0f7b2e48dfa1cb879ba0d5773aaa0b0f
Also, I saw there's a Windows section which seems to be outdated since there are no `.sln` files in the repo anymore. Not sure what are your plans with that.

Reply:
Thanks! I'll make a note to update the Windows instructions in a separate PR.